ajax/libs/exceljs/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "exceljs",
+  "version": "0.8.1",
+  "filename": "exceljs.min.js",
+  "description": "Excel Workbook Manager - Read and Write xlsx and csv Files.",
+  "license": "MIT",
+  "author": {
+    "name": "Guyon Roche"
+  },
```

Comment:
Let's modify the author field in one line, because there is only author name.
`"author": "Guyon Roche"`
The same with upstream https://github.com/guyonroche/exceljs/blob/master/package.json#L7
Thanks.

Reply:
well, maybe I would fix it tonight.